benchexec/tools/ceagle_absref.py

```diff
@@ -1,55 +0,0 @@
-# coding=utf-8
```

Comment:
Why remove this file? Again, it will make it difficult for other users to use this tool.